### DISCUSSION TITLE
Error using Deno

### DISCUSSION THREAD

**Discussion by aW4KeNiNG** [kobaltedev/kobalte#113] [Q&A]
Hi.

I'm trying to use Kobalte on a Deno project but I'm getting the next error:

```
error: 'import', and 'export' cannot be used outside of module code at file:///C:/Users/aw4ke/AppData/Local/deno/npm/registry.npmjs.org/@kobalte/core/0.6.0/dist/esm/index.js:1:1

  import { access, accessWith, getDocument, EventKey, getActiveElement, contains, ...

```

You can replicate the issue with the next code:

```typescript
import { Button } from 'npm:@kobalte/core';

console.log(typeof Button);
```

and ` deno run -A .\test.ts`

Any help is appreciated.

**Comment by fabien-ml** [comment]
Hi, I'm sorry but I have no experience with deno. 

You can ask the question on the SolidJS discord in the #kobalte-ui channel.
